KubeArmor/core/crioHandler.go

```diff
@@ -258,6 +258,31 @@ func (dm *KubeArmorDaemon) UpdateCrioContainer(ctx context.Context, containerID,
 			dm.RuntimeEnforcer.RegisterContainer(containerID, container.PidNS, container.MntNS)
 		}
 
+		if !dm.K8sEnabled {
```

Comment:
Can we remove this code duplication? Maybe use a common function call and use it from both crio and containerd handlers.